apps/testnorge-statisk-data-forvalter/src/main/java/no/nav/registre/sdforvalter/provider/rs/v2/StaticDataControllerV2.java

```diff
@@ -10,24 +11,41 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.RestController;
 
+import java.util.concurrent.TimeUnit;
+
 import no.nav.registre.sdforvalter.adapter.EregAdapter;
 import no.nav.registre.sdforvalter.domain.EregListe;
 
 @Slf4j
 @RestController
-@RequiredArgsConstructor
 @RequestMapping("/api/v1/faste-data")
 public class StaticDataControllerV2 {
+    public StaticDataControllerV2(
+            EregAdapter eregAdapter,
+            @Value("${controller.staticdata.cache.hours}") Integer caching
+    ) {
+        this.eregAdapter = eregAdapter;
+        this.caching = caching;
+    }
 
     private final EregAdapter eregAdapter;
+    private final Integer caching;
 
     @GetMapping("/ereg")
     public ResponseEntity<EregListe> getEregStaticData(@RequestParam(name = "gruppe", required = false) String gruppe) {
-        return ResponseEntity.ok(eregAdapter.fetchBy(gruppe));
```

Comment:
@langvatn Noen forskjell for konsumenter når body ligger i `ok()` kontra `.ok().body()`?

Reply:
ok(object x) retunerer ResponseEntity<object>

ok() retunerer ResponseEntity.Builder

ok().build() retunerer ResponseEntity<?>